src/com/team2502/robot2018/command/autonomous/ingredients/Paths.java

```diff
@@ -59,17 +61,33 @@
     public static class Left
     {
         public static final List<Waypoint> leftScale = Arrays.asList(
-                new Waypoint(new ImmutableVector2f(0, 0), 6),
-                new Waypoint(new ImmutableVector2f(-1.0F, 10.5F), 9F),
-                new Waypoint(new ImmutableVector2f(-1.0F, 22.5F), 9F),
-                new Waypoint(new ImmutableVector2f(-1.0F, 26.5F), 2F)
+                new Waypoint(new ImmutableVector2f(0, 0), 0, 20, -9),
+                new Waypoint(new ImmutableVector2f(-0.5F, 8.589F), 16F, 20, -7,true),
+                new Waypoint(new ImmutableVector2f(-0.5F, 12.405F), 5F, 20, -7,true,new RaiseElevatorScale()),
+                new Waypoint(new ImmutableVector2f(-0.5F, 20.5F), 3F, 20, -7, true),
+                new Waypoint(new ImmutableVector2f(2F, 23.0F), 0F, 20, -7) // max deceleration appears to be -7 ft / s^2
+                                                                    );
+
+        public static final List<Waypoint> leftScaleToSwitch = Arrays.asList(
+                new Waypoint(new ImmutableVector2f(1F, 21.2F), 0F, 20, -9), // max deceleration appears to be -7 ft / s^2
+                new Waypoint(new ImmutableVector2f(10F, 15F), 3F, 10, -9)
                                                                     );
 
         public static final List<Waypoint> leftSwitch = Arrays.asList(
-                new Waypoint(new ImmutableVector2f(0, 0), 6),
-                new Waypoint(new ImmutableVector2f(2F, 3), 9F),
-                new Waypoint(new ImmutableVector2f(4.67F, 7), 6F),
-                new Waypoint(new ImmutableVector2f(4.67F, 12), 2F)
+                new Waypoint(new ImmutableVector2f(0, 0), 0, 5, -5),
+                new Waypoint(new ImmutableVector2f(1.636F, 2.454F), 9F, 5, -5),
+                new Waypoint(new ImmutableVector2f(3.82006F, 5.726F), 6F, 5, -5),
```

Comment:
is the robot _really_ that accurate?

Reply:
that's what we want other teams to think 👌